pkg/apiserver/api_installer.go

```diff
@@ -125,13 +126,13 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	if err != nil {
 		return nil, err
 	}
-	versionedPtr, err := a.group.Creater.New(a.group.Version, kind)
+	versionedPtr, err := a.group.Creater.New(apiutil.GetGroupVersion(a.group.Group, a.group.Version), kind)
 	if err != nil {
 		return nil, err
 	}
 	versionedObject := indirectArbitraryPointer(versionedPtr)
 
-	mapping, err := a.group.Mapper.RESTMapping(kind, a.group.Version)
+	mapping, err := a.group.Mapper.RESTMapping(kind, apiutil.GetGroupVersion(a.group.Group, a.group.Version))
```

Comment:
I guess every implementation of RESTMapping needs its parameter name & documentation updated?